lib/archethic/bootstrap.ex

```diff
@@ -212,10 +214,44 @@ defmodule Archethic.Bootstrap do
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
 
+    if P2P.authorized_node?() do
+      resync_network_chain([:oracle, :node_shared_secrets], P2P.authorized_and_available_nodes())
+    end
+
     :persistent_term.put(:archethic_up, :up)
     Listener.listen()
   end
 
+  @spec resync_network_chain(list(atom), list(P2P.Node.t()) | []) :: :ok
+  def resync_network_chain(_type_list, _nodes = []),
+    do: Logger.info("Enforced Resync of Network Txs: failure, No-Nodes")
+
+  def resync_network_chain(type_list, nodes) when is_list(nodes) do
+    # spawn process for each type of txn to sync
+    Task.Supervisor.async_stream_nolink(Archethic.TaskSupervisor, type_list, fn type ->
+      # acquire a(any) address of type txn, (for querying DB)
+      with addr when is_binary(addr) <-
+             TransactionChain.list_addresses_by_type(type) |> Stream.take(1) |> Enum.at(0),
```

Comment:
Here yous should use the persistent term provided by PR #495 because for oracle, as there is multiple chain, `list_addresses_by_type` will not return you the last chain

Reply:
done!